src/zarr/abc/store.py

```diff
@@ -412,6 +412,18 @@ async def getsize_prefix(self, prefix: str) -> int:
         sizes = await concurrent_map(keys, self.getsize, limit=limit)
         return sum(sizes)
 
+    def _as_immutable(self: Self) -> Self:
+        """
+        Return a mutable copy of the store.
```

Comment:
It looks like perhaps the words "mutable" and "immutable" should be swapped between this method and the `_as_mutable` method, no?

Reply:
oops, good catch!